### ISSUE TITLE
website failed to be loaded when TM was applied

### ISSUE THREAD

**Issue by LambdaAlpha** [Tampermonkey/tampermonkey#1202]
(Please fill out the issue template with your details)

## Expected Behavior
<https://en.wikipedia.org/wiki/Random_Turing_machine> is loaded with TM scripts applied

## Actual Behavior
just white  empty content, nothing was loaded

but some time ago, it worked
recently I updated chrome and TM(auto updated by chrome), then it doesn't work now.

## Specifications

- Chromium: 89.0.4389.114
- TM: 4.12
- OS: ubuntu 20.04

## Script

(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name         fix some color
// @namespace    http://tampermonkey.net/
// @version      0.5
// @description  invert some color
// @author       LambdaAlpha
// @match        *://*.wikipedia.org/*
// @grant        GM_addStyle
// ==/UserScript==

GM_addStyle(`
.faster-single-canvas, .mwe-math-fallback-image-inline, .ztext img[eeimg="1"]{
  -webkit-filter: invert(1) !important;
  filter: invert(1) !important;
}
.bubble.bubble_primary {
    background-color: #3f51b5 !important;
}
`);
```

another

```js
// ==UserScript==
// @name     scroll bar style
// @include  *
// @grant    GM_addStyle
// @run-at   document-start
// ==/UserScript==

GM_addStyle(`
::-webkit-scrollbar {
  background: black !important;
}
::-webkit-scrollbar-thumb {
  background: #404040 !important;
}
` );
```

**Comment by derjanb** [comment]
This is working fine here. I'm also using the same configuration on a daily basis. 

Do you see errors at the extensions background console? Does a Chrome restart help? Are all other extensions disabled?

**Comment by LambdaAlpha** [comment]
I will add some more information.

- I disable TM in TM menu, then reload the page, the page was rendered properly
- I inspect the page, the html is like
  ```html
  <style id="xxx">
  some css
  <style id="xxxx">
  some css
  </style>
  </style>
  ```
- I logined in wikipedia and I have a custom css in that account,  at `https://en.wikipedia.org/wiki/User:xxx/common.css`
  ```css
  .mwe-math-fallback-image-inline {
      display: inline-block;
      vertical-align: middle;
      -webkit-filter: invert(1);
      filter: invert(1);
  }
  ```
- It works fine at my another pc, with nearly all the same configurations, eccept windows platform and TM version 4.11

**Comment by LambdaAlpha** [comment]
I add some more information.

- It occurs on some other websites, so it has nothing to do with wikipedia custom css.
- When it occurs, I disable the scroll bar style script and reload, then the bug is gone, I enable the script again and the bug doesn't occur again. This is the case on both two websites.

**Comment by LambdaAlpha** [comment]
Today my chrome in windows auto update TM, now I have the same problem as in ubuntu.

**Comment by LambdaAlpha** [comment]
I only saw one console log
```log
content.js:127 Uncaught TypeError: Failed to execute 'observe' on 'MutationObserver': parameter 1 is not of type 'Node'.
    at content.js:127
(anonymous) @ content.js:127
```

**Comment by LambdaAlpha** [comment]
I guess it has something to do with the `// @run-at   document-start`

**Comment by derjanb** [comment]
> content.js:127

Tampermonkey's content.js has no line 127 and also doesn't use MutationObservers. 
Can you please disable all other extensions and retry?

**Comment by LambdaAlpha** [comment]
I disable all other extensions and retry, the bug is still there. The error msg is gone.

![image](https://user-images.githubusercontent.com/20299598/114986857-1997ee80-9ec7-11eb-9848-f984fc537b79.png)
![image](https://user-images.githubusercontent.com/20299598/114986975-3c2a0780-9ec7-11eb-8108-deb3ce70ad59.png)

I refresh the page and it alternates between loading failed and loading success, i.e., loading failed, loading success, loading failed, loading success, ...

**BUT, I remove the `// @run-at document-start`, then the bug is gone.**

**Comment by derjanb** [comment]
I tried to fix the issue at Tampmonkey BETA 4.13.6135 which is review now.
If review takes too long, then please download the [crx](https://www.tampermonkey.net/crx/tampermonkey_beta_4_13_6135.crx) and drag and drop it to Chrome's extensions page (chrome://extensions).

**Comment by LambdaAlpha** [comment]
The bug didn't reproduce at Tampermonkey BETA 4.13.6135
